bin/forever

```diff
@@ -1,7 +1,15 @@
 #!/usr/bin/env zsh
 
+refresh_every="$1"
+shift
+
+if [ "$refresh_every" -eq "0" ]; then
 # This script runs a command and prompts the user.  If the user types
 # 'y' then exit.  If the user types anything else, loop.
+   local unused
+   while clear; date; "$@" ; ! read -q unused; do done
+   exit;
+fi
 
-local unused
-while clear; date; "$@" ; ! read -q unused; do done
+# This scrip runs a command every $refresh_every seconds.
```

Comment:
The comments here don't look right.  There should be a comment at the top explaining the script's signature (what it expects its arguments to be) and semantics.

Reply:
I tried to be more explicit in 502e622, does it look better ? :)